src/test/java/ui/RegisterTest.java

```diff
@@ -26,20 +27,29 @@ public RegisterTest() {
         this.password = getPropertyFromRegProp("password");
     }
 
-
+    public static String getPropertyFromRegProp(String prop){
+        return getPropertyValue(getPropertyFileContent(REG_FILE), prop);
+    }
 
     @BeforeMethod
     void beforeMethod() {
         this.registerPage = new RegisterPage(getDriver());
     }
 
+    @BeforeMethod
+    void beforeMethodForWordpress() {
+        this.wordpressPage = new WordpressPage(getDriver());
+    }
```

Comment:
Only one BeforeMethod will suffice. You can move the wordpressPage object instantiation into the  beforeMethod() method.

Reply:
done